packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -489,15 +459,9 @@ export interface IFluidDataStoreContext
 		createParam: CreateChildSummarizerNodeParam,
 	): CreateChildSummarizerNodeFn;
 
-	uploadBlob(blob: ArrayBufferLike, signal?: AbortSignal): Promise<IFluidHandle<ArrayBufferLike>>;
+	deleteChildSummarizerNode?(id: string): void;
```

Comment:
This makes me sad having to see that you need this.

Reply:
I wonder if it might be useful to move deleting of the child summarizer nodes to the parent?

Reply:
Another thought is can we just pass down the summarizerNode to the dataStoreContext from the runtime/parent. That way we don't need these weird looking create/getcreate functions.